packages/material-components-web/index.ts

```diff
@@ -38,6 +38,7 @@ import * as list from '@material/list/index';
 import * as menuSurface from '@material/menu-surface/index';
 import * as menu from '@material/menu/index';
 import * as notchedOutline from '@material/notched-outline/index';
+import * as progressIndicator from '@material/progress-indicator/index';
```

Comment:
I don't think this is necessary. We only use this internally.

Reply:
Should we follow convention of exporting every package, even ones that may not be used directly (i.e. base). Leaving it out would require adding some whitelists to the dependency test just for this package

Reply:
Updating the dependency test would be good. I'd prefer that over unnecessarily exporting the progress indicator interface.

Reply:
Done - let's continue further discussions in #5564